moped-editor/src/components/GridTable/GridTableSearch.js

```diff
@@ -324,9 +325,7 @@ const GridTableSearch = ({
               {query.config.showExport && (
                 <Button
                   disabled={
-                    projectsList &&
-                    !projectsList["project_list_view"].length &&
-                    true
+                    (parentData?.[query.config.table] ?? []).length === 0
```

Comment:
So the fix is quite simple, basically pass the name of the table in question to the array, then if provided calculate its length, and if it is zero then assume it's disabled.